### ISSUE TITLE
Publish Multi-Arch Operator Manifests

### ISSUE THREAD

**Issue by DWSR** [intel/intel-device-plugins-for-kubernetes#1306]
For those of us running mixed-arch clusters, it would be very nice to not have to constrain the operator to `amd64` nodes.

**Comment by mythi** [comment]
> For those of us running mixed-arch clusters, it would be very nice to not have to constrain the operator to `amd64` nodes.

just to make sure I understood: build images for non `amd64` archs too?

**Comment by DWSR** [comment]
Correct, yes. I glanced through the operator code and it doesn't seem to have anyhing amd64 specific. Given the increasing prevalance of mixed-arch clusters both on-prem and in the cloud, building for both arm64 and amd64 seems reasonable.
